SoftLayer/managers/ticket.py

```diff
@@ -39,7 +39,7 @@ def list_tickets(self, open_status=True, closed_status=True):
                 call = 'getClosedTickets'
             else:
                 raise ValueError("open_status and closed_status cannot both be False")
-        return self.client.call('Account', call, mask=mask, iter=True)
+        return self.client.call('Account', call, mask=mask, limit=100)
```

Comment:
I'd prefer we modified the `list_tickets` method to include parameters (with defaults) for `iter = False`, `limit=100`, `offset=0` so that way its easier for the CLI commands to pass in those parameters if needed.

Change looks good otherwise, thanks